src/Jira/Api/Client/CurlClient.php

```diff
@@ -82,12 +82,14 @@ public function sendRequest($method, $url, $data = array(), $endpoint, Authentic
             } else {
                 curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($data));
             }
-        } else {
-            if ($method == "PUT") {
-                curl_setopt($curl, CURLOPT_CUSTOMREQUEST, "PUT");
-                curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($data));
-            }
+        } elseif ($method == "DELETE") {
+            curl_setopt($curl, CURLOPT_CUSTOMREQUEST, "DELETE");
```

Comment:
The DELETE calls can have parameters. The correct implementation is done in #76.